uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/widget/grid/impl/BaseKeyboardOperation.java

```diff
@@ -72,7 +74,11 @@ protected boolean scrollSelectedCellIntoView(final GridWidget gridWidget) {
             isHeaderCellSelected = true;
         }
 
-        if (!(isGridWidgetRendered(gridWidget) || isGridColumnCandidateForScroll(gridWidget, isHeaderCellSelected))) {
+        final RenderingInformation renderingInformation = computeRenderingInformation(gridWidget);
```

Comment:
Now you obtain `RenderingInformation` here BUT it is only used in `isGridColumnCandidateForScroll(..)`.

So why not simply call `computeRenderingInformation(..)` _in_ `isGridColumnCandidateForScroll(..)`?

Reply:
will do, thank you